README.md

```diff
@@ -14,22 +13,30 @@ Terraform & Ansible are used to create and manage the nodes and deploy the appli
 
 ### Optional Steps: Deploying the VMs
 
-Steps 1-3 are only needed to create new EC2 instances to deploy the redpanda cluster on.
-
-If already created infrastructure is going to be used, they can safely be skipped but you will need to update the `hosts.ini` file with your specific information.
+To use existing infrastructure, update the `hosts.ini` file with the appropriate 
+jinformation. Otherwise see the :w
```

Comment:
vim ftw.
Let's remove the erroneous j and `:w` so we do not upset emacs users :laughing:

Reply:
Nice catch! Thanks! :)